elide-core/src/main/java/com/yahoo/elide/core/filter/InMemoryFilterOperation.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core.filter;
```

Comment:
Why is this in elide-core when the transaction the uses it is in the memory-store child project?